src/constants/api-urls.ts

```diff
@@ -0,0 +1,9 @@
+export const BASE_API_URL = '/api/v1/';
```

Comment:
we should have a `API_HOST_URL` too. That should be configurable using environment file